main.tf

```diff
@@ -12,11 +12,13 @@ module "metallb" {
   addon_config  = var.metallb_config
   addon_context = local.addon_context
 }
+
```

Comment:
Does the generator need to be updated to add these empty lines for future add-ons?

Reply:
They are already updated, this was previous